frontend/public/style/_common.scss

```diff
@@ -502,14 +502,6 @@ $co-external-link-padding-right: 15px;
   }
 }
 
-.co-m-loader {
```

Comment:
There are still a number of instances of `co-m-loader` throughout the code.
![Screen Shot 2019-07-26 at 2 38 26 PM](https://user-images.githubusercontent.com/895728/61973840-105c4500-afb3-11e9-8551-7371d8afbadc.png).
 
Related:  `.co-m-inline-loader` appears to be orphaned.

Reply:
Looks like that class name is still being used in `_common.scss L493` as well as in the e2e tests as a selector.